packages/client/src/modules/contact/containers/Contact.jsx

```diff
@@ -1,38 +1,34 @@
 import React from 'react';
-import { graphql, compose } from 'react-apollo';
+import { Mutation } from 'react-apollo';
 
 import ContactView from '../components/ContactView';
 
 import CONTACT from '../graphql/Contact.graphql';
 
 class Contact extends React.Component {
   render() {
-    return <ContactView {...this.props} />;
-  }
-}
+    return (
+      <Mutation mutation={CONTACT}>
+        {mutate => {
+          const addContact = async contactInput => {
+            try {
+              const contact = await mutate({ variables: { input: contactInput } });
 
-const ContactWithApollo = compose(
-  graphql(CONTACT, {
-    props: ({ mutate }) => ({
-      contact: async ({ name, email, content }) => {
-        try {
-          const {
-            data: { contact }
-          } = await mutate({
-            variables: { input: { name, email, content } }
-          });
+              if (contact.errors) {
+                return { errors: contact.errors };
+              }
 
-          if (contact.errors) {
-            return { errors: contact.errors };
-          }
+              return contact;
+            } catch (e) {
+              console.log('ERROR', e.graphQLErrors);
```

Comment:
I don't like this code, lets handle errors here somehow, lets show them after submit in the form as we do when we `login` for example.